packages/xlayers/src/app/core/sketch.service.ts

```diff
@@ -43,20 +35,20 @@ export interface ResourceImageData {
   providedIn: 'root'
 })
 export class SketchService {
-  _data: SketchData;
+  _data: SketchMSData;
 
   constructor(
     private sketchStyleParser: SketchStyleParserService,
     private http: HttpClient,
     private store: Store
   ) {
     this.store.select(UiState.currentFile).subscribe(currentFile => {
-      this._data = currentFile;
+      this._data = currentFile as any;
```

Comment:
why are you typecasting to `any`? It should be `SketchMSData`.

Reply:
Oh yes, i forgot to tag it. As I said, I didn't know if it was a good idea to put the resource field of SketchMSData in the `@type/sketchapp` since it is a xLayer client specific field.
So i had some type error.